server/src/test/java/com/broadcom/lsp/cobol/positive/CobolTextHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.broadcom.lsp.cobol.positive;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+@Slf4j
+@UtilityClass
+public class CobolTextHelper {
+    public CobolText fromStream(String name, InputStream stream) {
+        String fullText = null;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(stream));
+            StringBuilder builder = new StringBuilder();
+
+            for (String line = bufferedReader.readLine();
+                 line != null;
+                 line = bufferedReader.readLine()) {
+                builder.append(line).append("\r\n");
+            }
+            fullText = builder.toString();
```

Comment:
I hope that you can use this line here:
```
fullText = IOUtils.toString(stream, StandardCharsets.UTF_8.name());
```
We already have Apache IO library in our project.